recipes/tracy/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set version = "0.5.3" %}
-{% set sha256 = "5d9a056e96a77adec65a20d8e8191095174b2ec28907d3508b3a59482e5bf11e" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "b788ebd000065b8c0e376805bb9e1e16564c831f0a41da2eb119abade708e9d8" %}
 
 package:
   name: tracy
   version: '{{ version }}'
 
 source:
-  url: https://github.com/gear-genomics/tracy/archive/v{{ version }}.tar.gz
-  sha256: '{{ sha256 }}'
+  - url: https://github.com/gear-genomics/tracy/archive/v{{ version }}.tar.gz
+    sha256: '{{ sha256 }}'
+  - folder: src/xxsds
+    git_url: https://github.com/xxsds/sdsl-lite.git
+    git_rev: a654b57
```

Comment:
Does the sdsl-lite package not work for this?

Reply:
Sdsl-lite is GPL, SDSL v3 (github) is BSD so we could make tracy BSD licensed as well.

Reply:
Ah, gotcha, makes sense. :+1: